src/main/java/com/jhuguet/sb_taskv1/app/controller/GiftCertificateController.java

```diff
@@ -129,7 +131,9 @@ public GiftCertificate save(@RequestBody GiftCertificate giftCertificate) throws
 
     @PostMapping("/users/{userId}")
     public Order placeNewOrder(@RequestParam List<Integer> certificatesIds,
-                               @PathVariable(name = "userId") int userId) throws IdNotFound {
+                               @PathVariable(name = "userId") int userId,
```

Comment:
We can take userId from token.

Reply:
I'm validating user id passed in URL compared to the one in the jwt to secure the request

Reply:
I think we negotiated to use approach, when userId isn't passed , but is taken from token.

Reply:
It will be arranged